### PR TITLE
add page for non-merklized.md

### PR DESCRIPTION
**Pull request by vmidyllic**

### CODE REVIEW

File: mkdocs/docs/protocol/non-merklized.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
+# Non-merklized credentials
+
+## Motivation
+
+While Iden3 protocol allows to have a root of  merklized JSON-LD document for proving the inclusion of the credential entries in the tree, that increases level of privacy and flexibility for the credential data there is also a need to issue a core claim with a data itself. This is useful as for onchain issuers that can’t work with ld schemas in smart contracts and also for use case when the actual data must be saved, e.g. Auth BJJ credential public key representation.
```
Comment (OBrezhniev) [inline mkdocs/docs/protocol/non-merklized.md:5]: While Iden3 protocol allows to "merklize" the whole JSON-LD document (transform the document into key-value pairs and put them into a sparse merkle tree for later proving of credential field value in the tree), which removes limits on amount and structure of the credential data, there's still a need to issue a core claim with the data itself. This is particularly useful for onchain issuers, which can’t work with json-ld schemas in smart contracts (for many reasons), and also for use cases when the actual data must be saved for cheaper access inside circuits (e.g. Auth BJJ credential public key representation).

File: mkdocs/docs/protocol/non-merklized.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+other possible values:
+
+`slotValueA`  slotIndexB is value data slot  with index 6
+
+`slotValueB`  slotValueB is value data slot  with index 7
+
+See more information regarding data slot indexes [here](https://www.notion.so/Identity-Core-77fe2f04c8ad4e0296e2b90400d7ae3a?pvs=21). When user creates schema he should choose the slot to put the field.
+
+Nested structures are supported and path are created using concatenation with `.` e.g for birthday field in the Index Data Slot A mapping entry looks like that: `"...slotIndexA=passportInfo.birthday"`
```
Comment (OBrezhniev) [inline mkdocs/docs/protocol/non-merklized.md:62]: * "path *is* created"
* better show full value `"iden3:v1:slotIndexA=passportInfo.birthday"`

File: mkdocs/docs/protocol/non-merklized.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+              "birthday": {
+                "@type": "xsd:integer",
+                "@id": "vocab:birthday"
+              }
+            }
+          }
+      ...
+  ...
+}
+```
```
Comment (OBrezhniev) [inline mkdocs/docs/protocol/non-merklized.md:86]: Would be good to have full json-ld schema here as another example of NMC schema.

File: mkdocs/mkdocs.yml [hunk 4]
```diff
@@ -84,6 +84,7 @@ nav:
 #    - Libraries: "services/libraries.md"
     - Login protocol: "protocol/zklogin.md"
     - Proof Query language: "protocol/querylanguage.md"
+    - NON merklized credentials: "protocol/non-merklized.md"
```
Comment (OBrezhniev) [inline mkdocs/mkdocs.yml:87]: Non-merklized credentials

File: mkdocs/docs/protocol/non-merklized.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
+# Non-merklized credentials
+
+## Motivation
+
+While Iden3 protocol allows to have a root of  merklized JSON-LD document for proving the inclusion of the credential entries in the tree, that increases level of privacy and flexibility for the credential data there is also a need to issue a core claim with a data itself. This is useful as for onchain issuers that can’t work with ld schemas in smart contracts and also for use case when the actual data must be saved, e.g. Auth BJJ credential public key representation.
+
+**Core Concept**
+
+Non merklized credentials now can be created relying on JSON-LD schemas.
```
Comment (OBrezhniev) [inline mkdocs/docs/protocol/non-merklized.md:9]: Non-merklized credentials

File: mkdocs/docs/protocol/non-merklized.md (hunk 1 above)
Comment (vmidyllic) [reply mkdocs/docs/protocol/non-merklized.md]: fixed

File: mkdocs/docs/protocol/non-merklized.md (hunk 2 above)
Comment (vmidyllic) [reply mkdocs/docs/protocol/non-merklized.md]: fixed

File: mkdocs/docs/protocol/non-merklized.md (hunk 3 above)
Comment (vmidyllic) [reply mkdocs/docs/protocol/non-merklized.md:103]: fixed

File: mkdocs/mkdocs.yml (hunk 4 above)
Comment (vmidyllic) [reply mkdocs/mkdocs.yml]: fixed

File: mkdocs/docs/protocol/non-merklized.md (hunk 5 above)
Comment (vmidyllic) [reply mkdocs/docs/protocol/non-merklized.md]: fixed

**Review by OBrezhniev — APPROVED** [review]

**Pull request merged**